For some reason libtool 2.2.2 can't cope with this, LT_LANG(C++) works though.

RP - 14/4/08

Index: libusb-0.1.12/configure.in
===================================================================
--- libusb-0.1.12.orig/configure.in
+++ libusb-0.1.12/configure.in
@@ -149,7 +149,7 @@ if test "x$enable_debug" = "xyes"; then
 fi
 
 # Checks for programs.
-AC_LANG_CPLUSPLUS
+AC_PROG_CXX
 AC_PROG_CC
 AM_PROG_CC_C_O
 AC_PROG_INSTALL
Index: libusb-0.1.12/Makefile.am
===================================================================
--- libusb-0.1.12.orig/Makefile.am
+++ libusb-0.1.12/Makefile.am
@@ -4,7 +4,7 @@
 # gnu strictness chokes on README being autogenerated
 AUTOMAKE_OPTIONS = 1.4 foreign
 
-SUBDIRS = . tests doc
+SUBDIRS = . doc
 
 AM_CFLAGS = -Werror
 
